.isort.cfg

```diff
@@ -5,7 +5,4 @@ balanced_wrapping = true
 combine_as_imports = true
 case_sensitive = true
 
-known_first_party = kopf
-
-filter_files = true
```

Comment:
I remember that a couple weeks ago, `isort` was _sometimes_ treating Kopf itself as a third-party library. I didn't get when and why. But the explicit declaration did help.

So, this is better to keep it here.